### DISCUSSION TITLE
usage of react native to enhance security

### DISCUSSION THREAD

**Discussion by JohnRDOrazio** [kas-catholic/confessit-web#10] [Ideas]
Perhaps create-react-native-app could be implemented rather than creact-react-app, in order to make use of native smartphone technologies which would allow for further security in the app (such as fingerprint scanner / Touch ID).
https://www.npmjs.com/package/react-native-fingerprint-scanner

**Comment by kas-catholic** [comment]
An interesting idea. I'm not opposed to it, but I want to develop a clearer long-term plan before pursuing something like this. In particular, https://confessit.app is a website so it needs to support computers too. And while we could, for example, building it in two different formats (for web and native), there are already native solutions available that do this. (The [native ConfessIt Android app](https://play.google.com/store/apps/details?id=com.mikekasberg.confessit) already supports password protection on Android, and [Sabin's app](https://play.google.com/store/apps/details?id=catholic.confessionguide&hl=en_GB&gl=US) already supports this feature on Android or iOS.) Also, I think a lot of users don't share their phone and use a lock screen, so an additional password is somewhat redundant for a lot of users. Not to say that this isn't a useful feature idea, just that it isn't currently at the top of my priority list.
